### ISSUE TITLE
api.linkDevice failed always in whatsapp business

### ISSUE THREAD

**Issue by zhouao** [Auties00/Cobalt#447]
staying for a long time on the mobile, then current whatsapp crashed.
![x1](https://github.com/Auties00/Cobalt/assets/1142865/4854ea8b-f0a8-4c04-9052-d343c70cc23b)

**Comment by Auties00** [comment]
Yeah I still have work left for that api unfortunately, it will be complete before the next release

**Comment by zhouao** [comment]
that good. thank you very much!!!

**Comment by Auties00** [comment]
this could be a target for release 1.0 as the feature is complicated

**Comment by benlarsendk** [comment]
@Auties00 is this only for business, or is it the same for non-business accounts? I seem to be getting the same error on non-business. If there's anything we can do, let me know.

**Comment by Auties00** [comment]
I've decided to remove this feature as not enough people use it and it's very very hard to mantain
